dr_scaffold/scaffold_templates/model_templates.py

```diff
@@ -9,6 +9,10 @@
     class Meta:
         verbose_name_plural = "%s"
 
+    def __str__(self):
+        # return the name of field.
+        ...
```

Comment:
why the 3 dots ?

Reply:
Ideally, it would return the first text field. Or do you have any other suggestions?